Objects/typeobject.c

```diff
@@ -2880,14 +2880,19 @@ static const short slotoffsets[] = {
 };
 
 PyObject *
-PyType_FromSpecWithBases(PyType_Spec *spec, PyObject *bases)
+PyMetaType_FromSpecWithBases(PyTypeObject *meta_type, PyType_Spec *spec, PyObject *bases)
 {
-    return PyType_FromModuleAndSpec(NULL, spec, bases);
+    return PyMetaType_FromModuleAndSpec(NULL, meta_type, spec, bases);
 }
 
 PyObject *
-PyType_FromModuleAndSpec(PyObject *module, PyType_Spec *spec, PyObject *bases)
+PyMetaType_FromModuleAndSpec(PyObject *module, PyTypeObject *meta_type, PyType_Spec *spec, PyObject *bases)
```

Comment:
I propose the name PyMetaType_FromSpec and remove PyMetaType_FromSpecWithBases.

Reply:
So, you're suggesting renaming PyMetaType_FromSpecWithBases to just PyMetaType_FromSpec or just keeping the PyMetaType_FromSpec from above?

Reply:
Rename PyMetaType_FromModuleAndSpec (4 parameters) to PyMetaType_FromSpec. And so remove the other PyMetaType_FromSpec (2 parameters).

Reply:
Should I move the method out of "Py_LIMITED_API" then?